MediaBrowser.Providers/Manager/ProviderManager.cs

```diff
@@ -913,7 +913,8 @@ public void OnRefreshStart(BaseItem item)
             catch (Exception ex)
             {
                 // EventHandlers should never propagate exceptions, but we have little control over plugins...
-                _logger.LogError(ex, "Invoking {RefreshEvent} event handlers failed", nameof(RefreshStarted));
+                if (ex is not MissingMethodException)
```

Comment:
Ooops, this probably shouldn't have been included with this patch... it's there because the providers are throwing errors due to API changes... ignore this.